akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -102,12 +116,12 @@ object JdbcProjectionSpec {
 
     val envelopes =
       List(
-        Envelope(id, 1L, "abc"),
-        Envelope(id, 2L, "def"),
-        Envelope(id, 3L, "ghi"),
-        Envelope(id, 4L, "jkl"),
-        Envelope(id, 5L, "mno"),
-        Envelope(id, 6L, "pqr"))
+        Envelope(id, 1L, "e1"),
+        Envelope(id, 2L, "e2"),
+        Envelope(id, 3L, "e3"),
+        Envelope(id, 4L, "e4"),
+        Envelope(id, 5L, "e5"),
+        Envelope(id, 6L, "e6"))
```

Comment:
This is general improvement. Later, we should use this in the other tests. It's easier to follow the code if we say:

```scala
failWhen = _ == 4
with Skip strategy
projectedValueShouldBe(e1, e2, e3, e5, e6)
```

Otherwise we need to keed track that `ghi` is the fourth one and so on. Annoying.